### ISSUE TITLE
Timeshift not making scheduled snapshots

### ISSUE THREAD

**Issue by Aesyir** [teejee2008/timeshift#396]
Timeshift is not making scheduled snapshots, I set it to make daily snapshots and it does not do them, I checked /etc/cron.d/ and all I found was "timeshift-hourly" I tried "crontab -l" and it just said that this user has no crontab. I have cronie installed (arch linux) is there some other dependency I am missing? 

EDIT: I did realize timeshift always checks hourly to see when a snapshot is scheduled so that explains why there is only "timeshift-hourly". in the end though It is still not doing scheduled snapshots at all.

**Comment by BillFleming** [comment]
On my arch machines after some research I saw the schedules WERE being setup correctly, but the chron service wasn't enabled or running. As soon as I enabled it the scheduled snapshots started working. (I had installed from AUR.)
```
systemctl enable cronie.service 
systemctl start cronie.service 
```
Once you start it the jobs in /etc/cron.d should start running.
Next up is to check that the command is actually working correctly. After you setup timeshift if you run for example "sudo timeshift --check --scripted" from command line it should auto create a snapshot (with appropriate flags) if one is due. Then run the timeshift GUI afterwords to easily confirm the new snapshot is present. If this command doesn't work then the problem is somewhere in the settings for timeshift or user account/permission problem.
